NEWS.md

```diff
@@ -1,6 +1,10 @@
 
 # purrr 0.2.4.9000
 
+* `flatten_dfr` and `flatten_dfc` now aborts if `dplyr` is not installed. (#454)
```

Comment:
House style is to not put package names in back ticks, and to put `()` after function names. Can you please change?

Reply:
Oh sorry for that. I will definitely remember that for next time. 
And revise the tidyverse style guide.
Correction is done.